resotometrics/resotometrics/config.py

```diff
@@ -0,0 +1,66 @@
+import os
+from dataclasses import dataclass, field
+from typing import Dict, ClassVar, Optional
+from yaml import load
+
+try:
+    from yaml import CLoader as Loader
+except ImportError:
+    from yaml import Loader
+
+
+@dataclass
+class Metric:
+    kind: ClassVar[str] = "metric"
+    help: str = field(metadata={"description": "Metric help text"})
+    search: str = field(metadata={"description": "Aggregation search to run"})
+    type: str = field(metadata={"description": "Type of metric (gauge or counter)"})
+
+
+def _load_default_metrics() -> Dict[str, Metric]:
+    default_metrics = {}
+    local_path = os.path.abspath(os.path.dirname(__file__))
+    default_metrics_file = f"{local_path}/default_metrics.yaml"
```

Comment:
Can we add a test that ensures, that this file can be read?

Reply:
I mean, if the file can't be read it'll throw an appropriate exception. If the contents of the file are wrong/missformated, the core will fail the validation on "config save" since all of the file's content ends up in fields that are strictly typed.